From a9ef2fb87ca9a032d9fda62d5fa9882e07190acd Mon Sep 17 00:00:00 2001
From: Jakub Jelen <jjelen@redhat.com>
Date: Thu, 22 Oct 2026 13:10:15 +0200
Subject: [PATCH] Verify source signatures with openpgpverify

Replace the GnuPG-based %gpgverify with %openpgpverify, which uses
Sequoia sqv and takes the same options, as part of the Fedora 45 Change
https://fedoraproject.org/wiki/Changes/Sequoia_openpgpverify

Refresh B1D2BD1375BECB784CF4F8C4D73CF638C53C06BE from keyservers/WKD: same certificate(s), with self-signatures our copy was missing.
Release bumped with a changelog entry.

Assisted-by: Claude Opus 5.5
---
 B1D2BD1375BECB784CF4F8C4D73CF638C53C06BE | 330 +++++++++++++++++------
 libidn2.spec                             |   8 +-
 2 files changed, 248 insertions(+), 90 deletions(-)

diff --git a/B1D2BD1375BECB784CF4F8C4D73CF638C53C06BE b/B1D2BD1375BECB784CF4F8C4D73CF638C53C06BE
index e28c60c..f38aa3a 100644
--- a/B1D2BD1375BECB784CF4F8C4D73CF638C53C06BE
+++ b/B1D2BD1375BECB784CF4F8C4D73CF638C53C06BE
@@ -1,94 +1,248 @@
 -----BEGIN PGP PUBLIC KEY BLOCK-----
-Comment: B1D2 BD13 75BE CB78 4CF4  F8C4 D73C F638 C53C 06BE
-Comment: Simon Josefsson <simon@josefsson.org>
 
 xjMEXJLOtBYJKwYBBAHaRw8BAQdACIcrZIvhrxDBkK9fV+QlTmXxo2naObDuGtw5
 8YaxlOvNJVNpbW9uIEpvc2Vmc3NvbiA8c2ltb25Aam9zZWZzc29uLm9yZz7ClgQT
 FggAPgIbAwULCQgHAgYVCAkKCwIEFgIDAQIeAQIXgBYhBLHSvRN1vst4TPT4xNc8
-9jjFPAa+BQJiOJ5MBQkGwpOYAAoJENc89jjFPAa+OVQBAIltTRSXhbbx9WSt7RNz
-M4u/1zwc4Ihpqoexu5w42xQoAP937CR37xoIUTyoKSvNeXnJSVkUOeRiCImuEGkD
-3OZrAsKWBBMWCAA+AhsDBQsJCAcCBhUICQoLAgQWAgMBAh4BAheAFiEEsdK9E3W+
-y3hM9PjE1zz2OMU8Br4FAmD3wOsFCQXwdDcACgkQ1zz2OMU8Br6+0QD9GfOxBO7y
-V7CHuG8WPUrkHTEv9FmgPDHR1EoEPOY4B1wA/RcLfvKb0wZrr4I8h0W9MvaO533i
-BDN0TQr0KW0AH2kHwpYEExYIAD4CGwMFCwkIBwIGFQgJCgsCBBYCAwECHgECF4AW
-IQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUCX+IpbAUJBI5luAAKCRDXPPY4xTwGvtCJ
-AP0RI2UsT4uZawdEdyUz8yp82/gFflvugIQTX3lJo3iohAD/bFqoYSG0czpfe0lb
-oSz47MF+9KSs+/nx6oGMuAMewgbClgQTFggAPgIbAwULCQgHAgYVCAkKCwIEFgID
-AQIeAQIXgBYhBLHSvRN1vst4TPT4xNc89jjFPAa+BQJezg00BQkDekmAAAoJENc8
-9jjFPAa+7QMBAKyq5ZypvFOXgcwlNtQdf6F+SP9LnCNSreQRYo4RxSwAAQD7A+O5
-6xFPB1DIM74lpvaExNJFHbJXCIfFGifJycR0A8KWBBMWCAA+AhsDBQsJCAcCBhUI
-CQoLAgQWAgMBAh4BAheAFiEEsdK9E3W+y3hM9PjE1zz2OMU8Br4FAl2wR/oFCQI6
-PUYACgkQ1zz2OMU8Br6FEgEA7veg+Di9SEhfwG2qShdiKwVC9aZqp9gnLIIm6qLt
-fMYA/i0B5GsPglC7upw2a5qyTuSev7uaDLueBSV+mMq8cmAOwpYEExYIAD4WIQSx
-0r0Tdb7LeEz0+MTXPPY4xTwGvgUCXJLOtAIbAwUJARzEAAULCQgHAgYVCAkKCwIE
-FgIDAQIeAQIXgAAKCRDXPPY4xTwGvpqwAQDPG0khoyBLa4gASbXlKAPxgFiflZy6
-EeOJHOUJkMYSrAEAjXUcy1wrw72CADiG5UUXrbBnRSNM3VwPxMvzQNAfcgDOMwRc
-ks91FgkrBgEEAdpHDwEBB0C8whXBx6ys1Ujww2q2T2Kp/PR+Uz2scHDkYEYPuA2o
-BMJ+BBgWCAAmAhsgFiEEsdK9E3W+y3hM9PjE1zz2OMU8Br4FAmI4nlsFCQbCkuYA
-CgkQ1zz2OMU8Br6mnAD9HvUcBNa1S1G/33f7uhqn9JDHnGGqFoA2iIkdtoOO5mIA
-/1n/ThshksmDE5Isuea3X92eqSga3+JeJlrVneMnqa0Lwn4EGBYIACYCGyAWIQSx
-0r0Tdb7LeEz0+MTXPPY4xTwGvgUCYPfA9gUJBfBzgQAKCRDXPPY4xTwGvjLnAPoC
-iYpPmCcVKXoEmGBOFNcs+VzK/toEs6Vy0uFcaRJbiwEA/LLa/nZuEnjPRYIFCH7w
-41urLTtcC+SAW4KJyT8dQQ/CfgQYFggAJgIbIBYhBLHSvRN1vst4TPT4xNc89jjF
-PAa+BQJf4imCBQkEjmUNAAoJENc89jjFPAa+dtIA/0DjkqcuxttFDfSROXHQTLde
-0hJebfQSxuXlPu9cYBrfAP0YCUov9QgHqfxH6/nbw6tbdUqKkB1mBfWefZdjr4a3
-D8J+BBgWCAAmAhsgFiEEsdK9E3W+y3hM9PjE1zz2OMU8Br4FAl7ODUcFCQN6SNIA
-CgkQ1zz2OMU8Br4MQQEAsh6/JxnmqDgJoTzzhgzmzqGuvw6bPlgaZrxcPCf2eQAB
-ALRP6cqR4rDB5heHMuUv5B1cy2b38WkwJRqraGRh0oQGwn4EGBYIACYCGyAWIQSx
-0r0Tdb7LeEz0+MTXPPY4xTwGvgUCXbBIDQUJAjo8mAAKCRDXPPY4xTwGvu0SAQCK
-oSTVaP0+CerOAY5GTYs8A8tYiOo94pAWpA0FLOZzpwEA04K55KFRBu59exeb1yGH
-tXSJnQJCXrLnWOFpIwFQHADCfgQYFggAJhYhBLHSvRN1vst4TPT4xNc89jjFPAa+
-BQJcks91AhsgBQkBHMQAAAoJENc89jjFPAa+5JIA/35ATQ4NJVxFiyZIxfMQQ5oL
-CW2GHbYDUKZehXmzXK6bAP9pVSpU3VDSZITqJpvjtDYSSWwIFOXxxKr8DUNx7BAp
-CM4zBFySz4EWCSsGAQQB2kcPAQEHQOxTCIOaeXAxI2hIX4HK9bQTpNVei708oNr1
-Klm8qCGKwsA1BBgWCAAmAhsCFiEEsdK9E3W+y3hM9PjE1zz2OMU8Br4FAmI4nl8F
-CQbCkt4AgQkQ1zz2OMU8Br52IAQZFggAHRYhBKPMnIcLnTEKutTPL1FyKwj+R0Wi
-BQJcks+BAAoJEFFyKwj+R0Wip0wBAKAfjlqx5mAivAo69Q4D1B+yZ4TwZkdg24Ue
-Yz0URBLoAPoDjHlpi+hjqWj3ymC3DYc8OGZwJcUi2NcKtXgDQaYkDhASAP9Kz68w
-Qt3DEyIW4ktaI6eeysvM93dHwgs+p8/j8kic7QD/bXTcLqs0J+tfTj/tN4CW6PcJ
-kx3dDJfNvGAY38vVUgzCwDUEGBYIACYCGwIWIQSx0r0Tdb7LeEz0+MTXPPY4xTwG
-vgUCYPfA+gUJBfBzeQCBCRDXPPY4xTwGvnYgBBkWCAAdFiEEo8ychwudMQq61M8v
-UXIrCP5HRaIFAlySz4EACgkQUXIrCP5HRaKnTAEAoB+OWrHmYCK8Cjr1DgPUH7Jn
-hPBmR2DbhR5jPRREEugA+gOMeWmL6GOpaPfKYLcNhzw4ZnAlxSLY1wq1eANBpiQO
-JVIBAJjPGgVSATrdkZIeKqhDNgwBp2sHHiiZW7N0wEw64tm4AQCie9uKfAIE2h+L
-eGVWZ5lfqoXNvhoE/8YWsNVdysQ7DMLANQQYFggAJgIbAhYhBLHSvRN1vst4TPT4
-xNc89jjFPAa+BQJf4imHBQkEjmUGAIEJENc89jjFPAa+diAEGRYIAB0WIQSjzJyH
-C50xCrrUzy9RcisI/kdFogUCXJLPgQAKCRBRcisI/kdFoqdMAQCgH45aseZgIrwK
-OvUOA9QfsmeE8GZHYNuFHmM9FEQS6AD6A4x5aYvoY6lo98pgtw2HPDhmcCXFItjX
-CrV4A0GmJA5yiAEA6Ee/ptArOu2oGc2Okqf38MfN9q7SFTnzd5vutAI4Ib0BAKho
-dAz8j2HMUpxNVGjPHEwco7F1fkkR/78GJFsg+qADwsA1BBgWCAAmAhsCFiEEsdK9
-E3W+y3hM9PjE1zz2OMU8Br4FAl7ODUwFCQN6SMsAgQkQ1zz2OMU8Br52IAQZFggA
-HRYhBKPMnIcLnTEKutTPL1FyKwj+R0WiBQJcks+BAAoJEFFyKwj+R0Wip0wBAKAf
-jlqx5mAivAo69Q4D1B+yZ4TwZkdg24UeYz0URBLoAPoDjHlpi+hjqWj3ymC3DYc8
-OGZwJcUi2NcKtXgDQaYkDuQiAPwKnKAbzegaSATxN1cdFia4m80uJNFHMQL679WS
-BG3FIAEA8uLgxGud6SqFgIaFR4wrzrIgzVWqHxDuu56fJSf/iAfCwDUEGBYIACYC
-GwIWIQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUCXbBIEgUJAjo8kQCBCRDXPPY4xTwG
-vnYgBBkWCAAdFiEEo8ychwudMQq61M8vUXIrCP5HRaIFAlySz4EACgkQUXIrCP5H
-RaKnTAEAoB+OWrHmYCK8Cjr1DgPUH7JnhPBmR2DbhR5jPRREEugA+gOMeWmL6GOp
-aPfKYLcNhzw4ZnAlxSLY1wq1eANBpiQOte4A/2ju4S1HrcdnAVyd+BK5ieldpgOJ
-qKp9EFujf+zKuoKsAP4uq8+vBc6QmVri+ExSbqT/q5/pcUcBkxBVdda6G++yDMLA
-NQQYFggAJhYhBLHSvRN1vst4TPT4xNc89jjFPAa+BQJcks+BAhsCBQkBHMQAAIEJ
-ENc89jjFPAa+diAEGRYIAB0WIQSjzJyHC50xCrrUzy9RcisI/kdFogUCXJLPgQAK
-CRBRcisI/kdFoqdMAQCgH45aseZgIrwKOvUOA9QfsmeE8GZHYNuFHmM9FEQS6AD6
-A4x5aYvoY6lo98pgtw2HPDhmcCXFItjXCrV4A0GmJA4KLgEAuT/fZL+h/jy/QW9y
-fSi4BT56PGU7n/MoUzjOwG8flj4BAOCm+fqLhrM0SQzzjA0qdCrsTK/7rBSZ908t
-bNYOawgOzjgEXJLPahIKKwYBBAGXVQEFAQEHQDGVG6YNbqdljsD5UJ96T1WkTHo8
-mZ0nJg+3eU0cx3ocAwEIB8J+BBgWCAAmAhsMFiEEsdK9E3W+y3hM9PjE1zz2OMU8
-Br4FAmI4nlQFCQbCkuoACgkQ1zz2OMU8Br7QXgEAhd81O+aan2sLslGjC749qnaq
-AHwan9Nxz8f+J9SzG3UBAPTMOR4aoC+u8KxUxkNU53Ccf+bgGHY4OJ0gTDLLpPIG
-wn4EGBYIACYCGwwWIQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUCYPfA8QUJBfBzhwAK
-CRDXPPY4xTwGvtQ/APwP8XxDUEogiXWIatg4A8oap242CJQjPf9IrwETGf7ckAEA
-oF/XCoJNOsSKRGPgTnVA7vD8tNQpv6BZsk4WB3ShOAfCfgQYFggAJgIbDBYhBLHS
-vRN1vst4TPT4xNc89jjFPAa+BQJf4il7BQkEjmURAAoJENc89jjFPAa+410BAOK0
-ISCDC4Rd/LLUBMSfZsLIpI1yuFnDPp+OAc0Ec9tkAQDJ1Jz/XlxCwf/gYoqTLXkZ
-xGzd33QWjH2JKVZyKNF2CsJ+BBgWCAAmAhsMFiEEsdK9E3W+y3hM9PjE1zz2OMU8
-Br4FAl7ODUAFCQN6SNYACgkQ1zz2OMU8Br7vEAD/S59d+oS7f8i38eCmIwcTwHX3
-faEWHd8alhQcTWZp/gYA/12a4TWmNzBAVeMiMeLbOiU+3SX1xnGzY8StU9gZV3UB
-wn4EGBYIACYCGwwWIQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUCXbBIBgUJAjo8nAAK
-CRDXPPY4xTwGvuhAAQDvbSbrxBwmqvbEwJuprPVn9Nx/bwfWKfbS2Dqg/huOhAD9
-E1YBmxr5qiAbFJKP0O7Sj22HXX7LiALyf8eORehJPAjCfgQYFggAJhYhBLHSvRN1
-vst4TPT4xNc89jjFPAa+BQJcks9qAhsMBQkBHMQAAAoJENc89jjFPAa+4kMBAILG
-9rwrhcPSAvay4XDgJ8SqDHm8AsKA3E7zirxVq5X9AQCNlXB+RWh7yX61utkJrziB
-qQti/j3vHKX6c3yU4FLNCw==
-=KvZq
+9jjFPAa+BQJp4fWRBQkOa+rdAAoJENc89jjFPAa+hWIA/1lQvrJeGlQq50lP6tm9
+9D1zDy7J1tQ3ha4x0Jx7rkFTAP9hpUKuTvm6m1fXyiZVYZlu2+Id/Dq3CIAZvNF+
+XEr2BMKWBBMWCAA+AhsDBQsJCAcCBhUICQoLAgQWAgMBAh4BAheAFiEEsdK9E3W+
+y3hM9PjE1zz2OMU8Br4FAmfRdCQFCQ1kZvAACgkQ1zz2OMU8Br4G0gD+JHvcJ8G6
+sb3LykBKHdxsY4wekNl8wC2iy7qNuk7r0YcA/inuXnt36JW3LZVAyzH7dMJIC9mS
+mvg/8W5PNnwll7wGwpYEExYIAD4CGwMFCwkIBwIGFQgJCgsCBBYCAwECHgECF4AW
+IQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUCZf2ICAUJC3oRVAAKCRDXPPY4xTwGvgm6
+APsFGtOkQ+XuVUwsb0lrT0u+VqtqgXmBmZ39I71DMWtp+AD+MZFHgDz5zbjLE5tn
+7BQgyBmhDNmJ+lW9obs809RcPwTClgQTFggAPgIbAwULCQgHAgYVCAkKCwIEFgID
+AQIeAQIXgBYhBLHSvRN1vst4TPT4xNc89jjFPAa+BQJk8nONBQkJfGjZAAoJENc8
+9jjFPAa+698A/ifBlgu1rdSn4koQ6E8Ez1XhfZqZMAzbNW8DoQ2BVSVtAP9bTdYt
+tIE1j5pK89u0+JMxcmvh8H3P3ntMrEWp2ePzBcKWBBMWCAA+AhsDBQsJCAcCBhUI
+CQoLAgQWAgMBAh4BAheAFiEEsdK9E3W+y3hM9PjE1zz2OMU8Br4FAmM8VIYFCQh2
+8tIACgkQ1zz2OMU8Br7VIAEAnfGK0ZHyZkgGjpBqws7dopcgmxnEEcsUFKBdwYR+
+jqMBALXrCpRKBONznhae0SEpy7G+MSp7JmXNKSWBo3zIzxcNwpYEExYIAD4CGwMF
+CwkIBwIGFQgJCgsCBBYCAwECHgECF4AWIQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUC
+YjieTAUJBsKTmAAKCRDXPPY4xTwGvjlUAQCJbU0Ul4W28fVkre0TczOLv9c8HOCI
+aaqHsbucONsUKAD/d+wkd+8aCFE8qCkrzXl5yUlZFDnkYgiJrhBpA9zmawLCrQQT
+FggAPgIbAwULCQgHAgYVCAkKCwIEFgIDAQIeAQIXgBYhBLHSvRN1vst4TPT4xNc8
+9jjFPAa+BQJg98DrBQkF8HQ3ACEJENc89jjFPAa+FiEEsdK9E3W+y3hM9PjE1zz2
+OMU8Br6+0QD9GfOxBO7yV7CHuG8WPUrkHTEv9FmgPDHR1EoEPOY4B1wA/RcLfvKb
+0wZrr4I8h0W9MvaO533iBDN0TQr0KW0AH2kHwpYEExYIAD4CGwMFCwkIBwIGFQgJ
+CgsCBBYCAwECHgECF4AWIQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUCX+IpbAUJBI5l
+uAAKCRDXPPY4xTwGvtCJAP0RI2UsT4uZawdEdyUz8yp82/gFflvugIQTX3lJo3io
+hAD/bFqoYSG0czpfe0lboSz47MF+9KSs+/nx6oGMuAMewgbClgQTFggAPgIbAwUL
+CQgHAgYVCAkKCwIEFgIDAQIeAQIXgBYhBLHSvRN1vst4TPT4xNc89jjFPAa+BQJe
+zg00BQkDekmAAAoJENc89jjFPAa+7QMBAKyq5ZypvFOXgcwlNtQdf6F+SP9LnCNS
+reQRYo4RxSwAAQD7A+O56xFPB1DIM74lpvaExNJFHbJXCIfFGifJycR0A8KWBBMW
+CAA+AhsDBQsJCAcCBhUICQoLAgQWAgMBAh4BAheAFiEEsdK9E3W+y3hM9PjE1zz2
+OMU8Br4FAl2wR/oFCQI6PUYACgkQ1zz2OMU8Br6FEgEA7veg+Di9SEhfwG2qShdi
+KwVC9aZqp9gnLIIm6qLtfMYA/i0B5GsPglC7upw2a5qyTuSev7uaDLueBSV+mMq8
+cmAOwpYEExYIAD4WIQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUCXJLOtAIbAwUJARzE
+AAULCQgHAgYVCAkKCwIEFgIDAQIeAQIXgAAKCRDXPPY4xTwGvpqwAQDPG0khoyBL
+a4gASbXlKAPxgFiflZy6EeOJHOUJkMYSrAEAjXUcy1wrw72CADiG5UUXrbBnRSNM
+3VwPxMvzQNAfcgDCdQQQFggAHRYhBE0UBQZTpALXNocEnSQEyVRuFFNgBQJmtrmp
+AAoJECQEyVRuFFNg6c0BALRpGTWygLHelMjwDaFaI/78/oXzDTskNAEMeFayn8/2
+APwNKnsSeLNo3PhMNP/AIn1oMCtHbjMiguErhbEQDrVWCcLBcwQQAQgAHRYhBK5I
+wPWbGZigYpKmmZa9T/ZOcJ/eBQJkECSsAAoJEJa9T/ZOcJ/eK/QP/iIaGNLuOd8P
+HnTzxWabo6XHvsWfY3WGWeb0yZiTozW0vBC1vc9hRJBQgSK0nhM/tjT1do70npWz
+KExC1BSLOHQki0zbSW/FZdmDEYhYfFrSQ0QnO6onxg7fkE5J0AVDqBVWKJcKIHB1
+zMfr3hNHBJLcYZRvKSEbOK0VdHZSZr6rkOxdqcLKvneteGfX87FBysyc5vWc+0nG
+Bh18+TcOwzMDNaqdJzc02wK28d7vbOwL2AV2xLYSWXnwqUcWAa6GCcwhkemZVoMn
+XfaJ8EYE6Qd1tPWv9BWy/V7K9WQW2ywK1kMesC+jUN3/8BytxJ+fIgruNVHmXh5h
+VWJJXA3uC9Sv2TgvW/JhMuQxpZnt3n5yS8llIlVoio2DZ+iheQLdi3LZ3dPFb0hk
+cWN0KS6WOVXA4zUxFUhXqiBp0ZiV9+3rFBWH/SulwnEHkrX+/IyvUchfXWfyQwSg
+sUS6ITVnpdrGjiXeQBZswPq4m9ESQ28cZzfjOif21uubbFmoHyyiTKtlVhqUxfzB
+gMa+WKJcVw+nlQ9p4HKIdMmrpOwo9XNl89tvtIq4bgw0vjyva7NiNu9gQb4D52La
+0ktDCdEE1yu01KYMMXNghMy3Y/eJJ80T2X690hgP1WHkawwgKRWJFY5q4L0Di3Lk
+BUFdyV3752bADmiPcme+5OrUWueTsljPwsFzBBABCAAdFiEERiIlw7RvNIefyEls
+1gWEjtfmmHEFAmNOVnIACgkQ1gWEjtfmmHFNRg//Stnf8AU8n1iyZTpk7MsbsvsO
+oRiyWFfwoKVukTBQRDNq2c4Mb2c807mF5mE/USMX/DyLrS1HrUeIF4/VkPBphm+I
++roeJ1NxTfR6ukyEH3mdZ98GGBXR+sA7h95YJYWZAVdhHcaH+/LnobvsgA4PIahH
++tRJZ1ZF3bo+xmad6pUq/9evlxvNd8WebmViFG0KuimhUst8q2BL7z6jLy44zB7B
+UqAvYKkwkRJoacRk8uIcXLDpjQN99SHHaKdjRNZWIOTXOkRGsROshH95cL14E1uq
+KIRHY7SPYMuQ+BBLbmQV1FjqHqCCQ7IADMQshLZxPUAK4L5UgSmYaPImGkRIDQQr
+OwtWjS8P1amzBjhjeB3ok2jsBjX38A+YFpYeBlcPdupv/ud/kdWRlgxlL56O4+qN
+5szr9dehfM6CjcqVLwBoWGVm8uNKERtGZvX3bvk0WiTHAi1VqinsNyn0hiFuO3Bc
+NrBkW3SWN00L9W+JkJK4OV+99pOOg1CoUDC0Zayt7zz1KgmHWFf/R6d/tN1kcqX3
+UtQPrnQRALtDF2V6aOtOgXQpt4NQfBRkvNlyJW/QtbcSTM1h2EW9laG7nz59Le0O
+Pr7WqPZMXxdMXnhmcTsn4mKl9GRc2mtp0OblcpJpftWmB4wmPdrWPnoghvwCAquo
+0eZcO2y/GZi3WlqnvoLCwXMEEAEKAB0WIQSPiDgd1chz5K5IfaUZm/EkNjIEagUC
+Yzv9hgAKCRAZm/EkNjIEagdXEACP4qlQiBSNf4CAk0xeEyxaez7s8aI48STFpyR2
+D1ePwgP17Vdwx8TOUpBMvTl7CoPnep1kUGVRehLfOhP4vxGmMNKeVhbWQJ3Z1cBQ
+5gEUUG8Ojti7CEr3ObbWsZIdOjFc8k9UwrWqgvRKGGpm8c/QSxIe2+CEOYvcPF5d
++YIeOPRFwbG7Oyrv4eqcGcg+QNcn2RicmhNsOoSuEV6IpXYMdZ5jbc5dMzG2IRGw
+8zB0Top7PCJqheFi2lkYgGkOZRqLILZvp9KygUXjuM1jA48xWLtnSyfyt/OL4cR2
+iJ0DbLAq3V82KGJoP/UEiamqRR/vLo/osKvRxOrMfoDV/FX78lhC6DArQH96Lg15
+95dhOGumI4F/RUWoK9e5vF95KlKQhYfzah0kEUkWQpqMABfAD0xeMCQOKsJcdJ2q
+qOgyEdED3OVnU4qAd3x29INd3Vxxl6XpvuRPOz88FvkcRAAqwuy3DdLJyhO7bgHk
+5FYPxdL3BFz3915dhLO0KkvAKIOrC0vEw0Rh4CZKAJYyBEHlRnk0J38/FmfGiLJm
+EeOnH3Cl9liubhBZ8ETduBCfkzI02gsEse1I60iatwXlUD3dJYUWfvABZ4pHuzlP
+IjZXSsewNWwFR0cXrQNwBL3kZLH6doJLGVZWnquDQSy7cK+aYHXus02Ov3r1Uv93
+TZPdZsLBRwQQAQoAHRYhBJqpvbEbsbmaIShaMwZkp2lUJl6MBQJgrTXfAAoJEAZk
+p2lUJl6M0+AOnRciAYVTSuUmJmyB4Q06wImbV5vkVDNPj/Jc8qlNgn9+HcgTa/a6
+mLjJuhp5HZJLYtVH2C4brPgnLGNPHVbeYf6v9gtwb43r6HvAbgvqGBg1nBHL/ua4
+b4WFRCs0woilx9j9CysVx29sV+hSe1EH9pcU8m6SfLh+0GobYuGpPSrvfvA88LRe
+qev/gViqEPQ91dm4RlDBvY91B2fSLVsD7eMb3pao7Cancg4ye0BqtpgzP+28GgnT
+7od4YL/kXUd6+c1EfKqNplBBd2a69F2z3j3a7tOtNw6y2jldRuJEoA3/hv1bW8pM
+qoLLxkVePK3IQxrvzsl7ue6qSrGNgiw3QKTqs7W7yio/kEiON94E2SHvkEVKNXfB
+DDHz613D05d4OlpEVzA7YffNvCLZHAw6rNmSiq3+82G7xPpfhjy7saZe+ugekDz9
+ccbVm+xDVuqM2K2mgtwQ1QHBUtvYbL36sTMHBWmmAk0MlbftD9TFJ1eaE2Tn4uzy
+/3nxwYOa6y1eepHjcmEUi5RBqKbIjX+qse50onWE3AHwjxsl4MPZ6aSd849qqTnq
+IQ7L8IqjhQRY+pWEs+bKc7aLAMpPuQPR/6rd9OIDzwqGHScUslI7P+Zqr4RDdsLB
+cwQQAQoAHRYhBHY2L9O8q5d3Km3BWsGO5orhkNC8BQJffafzAAoJEMGO5orhkNC8
+LeUP/0l218MZJVfKKfB9uU1YnzfrxQiZeUFOp6YvdVX2U42Z5BOSUSFS+vTKK9nD
+Bwsi9x30SQpXrmyepRPSS6e9U45OUIcDU7yuGtnuLhvEdVbUzvJC2kSzB0ETydHt
+wbk6M3Rrk86mBols3k+rYn/LcKowi+/PxFbUbHZQBrEJdtpLw345W0eMb2hH37ep
+fR3mFjkWanzGzuKCWOUIJ/8oaIUxdwwruJetknupb34cWUsvIwv9oR8HGW26zTYx
+Ymj85yhB90m3/q5I5zRXvtOmSP9tZDaUZbrn3mczBr4/QSRQvm6UWxF4fwHgPqGQ
+OOpU10iY5p7uwmF3YGkIiox+JlUaQIjfnt33rXIrvZnlFkF/uUzRYpHhRAdoHiH/
+O0DUT7fw237RwtKoxepvOZTJX8qatzyiHkliqjmoeznT/SkTh9TuDDZRMegJjA6X
+jbltL+B2DLEzbAj7uPHtQ2J16gYtl1rzwwbdI8ofKyDitJ+qZh0rrQ+7QriMzpGm
+W9odBgiH/YsdXpFzwuFc2sfwGa1FxEMKJGS91Xs2FZXFaFUJ0nwC2SolJYy855i0
+jlnhv6g4JArihXRaVC379fbZmwp80PuP1WifWB5bbOkMkNsuiJnVGWx/ki6LRC4L
+a7JoCTX4trG2JkSHybi3FTveafMZSGBU3IszPbCi9uqIKGO9wsFzBBMBCAAdFiEE
+YgqkIrEtHh218zwyGQ5faz9Xw4QFAl97chkACgkQGQ5faz9Xw4QqpBAAkXoWAj8U
+uPchVuEDt5y8s6aedpbte//Rs7aPrNKuR77wrTV1ctQox95MOk9shdldfxfGAExI
+c7zNdGfmmQLhL2WMJS5wnx/UQ0WwDqTKwD5VhCnM+9u7Ojv3Q8RpO3jBT5Jo4tz8
+Dq2RNyvZUCDtgsmo8/o/tcXRPw0WaH00sCOIy2rqU6jfefnDsuNcLgDW8uMHOsOh
+Rvz/qCKaUEAeKdbO6VeYXzwHTVS2u5iwFguQAbhqCSpISoCl5z9TLFBMpD9MvtVp
+Azsw49TMMIm492fSqwWQTMVLh94mh0X5+uGrHaHz13vjsxPZN0a8JtE851LMLvzS
+IiCAy4EvQLpWEQHZm6qxEHGaKsH1C0m//4rm+2MF0mrfJDUXRyA+IqAh8THUqEjk
+5sx645/lNMLSQEKRXx5+DqFj7gV1roRNNsNXm/MusjD7i3HUC0BkFsn+Qs5g/TpI
+qDOYiEHVwf8VsbKe8D2+yoacxfybIHTCIm6W1i5akNwsAwLT6KHGrBD4utdS2HOq
+fD+AdHRMYUaz7tQh0cujF4gBTvG2gLrSaE1Lzs3DKRM0gOqvUUhPqNZLYxetMUN0
+0HexNx1tipdmcbMb9Qe/1MuhwuCXtnchDuBUBxvCe4RRr1Zr7bPh3+ryNorJOfFT
+Ay75ZOesE5VjYA2w1jO7uh/GlXSgXoWeRA7CwUcEEAEKAB0WIQSaqb2xG7G5miEo
+WjMGZKdpVCZejAUCXJPswQAKCRAGZKdpVCZejECuDp0UImvoT6T3p0mdFDED7jHT
+ItFO+1TVhmLFqM+HYlRJ4tV+6eqvS6eka5zMj9agFbr4Mswg5iqMN5xPr0etuhIe
+4qp8Oi94sFmI9F4PHm2bJvJm9x3ZrqG4TEqKrnS5ZcZszVYiM6UlDOBsrrzytIXd
+/yOhTugjVJJU/De1rAx80gj75nWgzMs1UhgDVpea5311v6jxfbAC0z9OyUl1hTfE
+gjqyQl0jxgFjhdIknkR23MC2oYKH/Z2NdQ+znsDb0ZYrYP9AxJI4Slien2NZOUfK
+8h1rUCbAeIyerqvbbiehCn3PwPOXgSY6SXSaXvQFvkQOBRzxBXHyoQeMVgdSdhwf
+9KHLOZupXltVriF8w3PVgfejSJEkFrbJ6lH8Yp+ON/6yH+Zrkt+9xso5kfO1cw8f
+FeiyIrUqAwdMgWjVufo8Jbey6LZ8GUHhJ0ZEuFQ5M+I4dtl83Qm5qZnasgg4Ewrs
+goXw6dhIwxYPodm1d8RLw2SO68deYJHwfTw6pFm31ARN1ZGpEFg5UJjbONPRd3iU
+QfJGhlZaKboGUtIj8MdRcKPre/MqGvNuFTGrxwqcNpOUMGa/i9mPXvN8wITln67P
+VOsfd2+I/jMQ1yDkECbvURMhQkLOMwRcks91FgkrBgEEAdpHDwEBB0C8whXBx6ys
+1Ujww2q2T2Kp/PR+Uz2scHDkYEYPuA2oBMJ+BBgWCAAmAhsgFiEEsdK9E3W+y3hM
+9PjE1zz2OMU8Br4FAmngltcFCQ5qi2IACgkQ1zz2OMU8Br62MwEAwZbs4Y/YigDX
+eG8Pw+9alVkDJq8aOnY75Rb7pt3mrXMA/29JFumgWJi0pmXQGMXV8mN4xvioQUxw
+O8/rAvOEOMMAwn4EGBYIACYCGyAWIQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUCZ9F0
+NQUJDWRmQAAKCRDXPPY4xTwGvo+xAQCmGnjeb/lNgmAEw4u7HjJatR6/U7JzdisJ
+Kf0gRO5migEA8kAlMeg9/YSy5YtlUZ41awS6WAboJiXpdvn837Vs/gPCfgQYFggA
+JgIbIBYhBLHSvRN1vst4TPT4xNc89jjFPAa+BQJl/YgkBQkLehCvAAoJENc89jjF
+PAa+jEIBAJ7+IxBm8dmZJB5/lGcvUc5bLTc4e2xnKEYudCOzHs+EAQDc7YrR01s9
+baJkxyf0xcA52Z9/fOR82tYRaRlH/HleDMJ+BBgWCAAmAhsgFiEEsdK9E3W+y3hM
+9PjE1zz2OMU8Br4FAmTyc5cFCQl8aCIACgkQ1zz2OMU8Br4nzwD/f5iWiVpnpc6W
+BFAB+sX6S/Jpy2iBoyz/qHlBcF3Ji9MA/0HBZLuo3C7T90GSu+FSIfg5VofnfeES
+3SNvtncX0QQJwn4EGBYIACYCGyAWIQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUCYzxU
+lAUJCHbyHwAKCRDXPPY4xTwGvg2wAQDDNZ4Gzqn+VszD046QpEuU0lsWkx4LMxaU
+RhWfNSQTUAD/e+r8Dt8Tlr4ZpTYfNiSztsqvZlBVFzFBBp2Kw/AYDgjCfgQYFggA
+JgIbIBYhBLHSvRN1vst4TPT4xNc89jjFPAa+BQJiOJ5bBQkGwpLmAAoJENc89jjF
+PAa+ppwA/R71HATWtUtRv993+7oap/SQx5xhqhaANoiJHbaDjuZiAP9Z/04bIZLJ
+gxOSLLnmt1/dnqkoGt/iXiZa1Z3jJ6mtC8KVBBgWCAAmAhsgFiEEsdK9E3W+y3hM
+9PjE1zz2OMU8Br4FAmD3wPYFCQXwc4EAIQkQ1zz2OMU8Br4WIQSx0r0Tdb7LeEz0
++MTXPPY4xTwGvjLnAPoCiYpPmCcVKXoEmGBOFNcs+VzK/toEs6Vy0uFcaRJbiwEA
+/LLa/nZuEnjPRYIFCH7w41urLTtcC+SAW4KJyT8dQQ/CfgQYFggAJgIbIBYhBLHS
+vRN1vst4TPT4xNc89jjFPAa+BQJf4imCBQkEjmUNAAoJENc89jjFPAa+dtIA/0Dj
+kqcuxttFDfSROXHQTLde0hJebfQSxuXlPu9cYBrfAP0YCUov9QgHqfxH6/nbw6tb
+dUqKkB1mBfWefZdjr4a3D8J+BBgWCAAmAhsgFiEEsdK9E3W+y3hM9PjE1zz2OMU8
+Br4FAl7ODUcFCQN6SNIACgkQ1zz2OMU8Br4MQQEAsh6/JxnmqDgJoTzzhgzmzqGu
+vw6bPlgaZrxcPCf2eQABALRP6cqR4rDB5heHMuUv5B1cy2b38WkwJRqraGRh0oQG
+wn4EGBYIACYCGyAWIQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUCXbBIDQUJAjo8mAAK
+CRDXPPY4xTwGvu0SAQCKoSTVaP0+CerOAY5GTYs8A8tYiOo94pAWpA0FLOZzpwEA
+04K55KFRBu59exeb1yGHtXSJnQJCXrLnWOFpIwFQHADCfgQYFggAJhYhBLHSvRN1
+vst4TPT4xNc89jjFPAa+BQJcks91AhsgBQkBHMQAAAoJENc89jjFPAa+5JIA/35A
+TQ4NJVxFiyZIxfMQQ5oLCW2GHbYDUKZehXmzXK6bAP9pVSpU3VDSZITqJpvjtDYS
+SWwIFOXxxKr8DUNx7BApCM4zBFySz4EWCSsGAQQB2kcPAQEHQOxTCIOaeXAxI2hI
+X4HK9bQTpNVei708oNr1Klm8qCGKwsA1BBgWCAAmAhsCFiEEsdK9E3W+y3hM9PjE
+1zz2OMU8Br4FAmngltcFCQ5qi1YAgQkQ1zz2OMU8Br52IAQZFggAHRYhBKPMnIcL
+nTEKutTPL1FyKwj+R0WiBQJcks+BAAoJEFFyKwj+R0Wip0wBAKAfjlqx5mAivAo6
+9Q4D1B+yZ4TwZkdg24UeYz0URBLoAPoDjHlpi+hjqWj3ymC3DYc8OGZwJcUi2NcK
+tXgDQaYkDrOwAPoCCB6QQKQ6XA49bCocw54Jqb8EH9FZB5nurPhBu6koKQEA3v1z
+K1NHND2t44vOXDKmYb8yThSP7KHuutpcy9EGNwXCwDUEGBYIACYCGwIWIQSx0r0T
+db7LeEz0+MTXPPY4xTwGvgUCZ9F0SgUJDWRmSQCBCRDXPPY4xTwGvnYgBBkWCAAd
+FiEEo8ychwudMQq61M8vUXIrCP5HRaIFAlySz4EACgkQUXIrCP5HRaKnTAEAoB+O
+WrHmYCK8Cjr1DgPUH7JnhPBmR2DbhR5jPRREEugA+gOMeWmL6GOpaPfKYLcNhzw4
+ZnAlxSLY1wq1eANBpiQOwUUBAO64fbZek6FPlRK0DrlWsrjCXuLi6PUxyzCAY6lG
+2nhUAQC6qobB9mkZlZ0qihy1x4JRtflqFcqqT9n7iUZkCDIiDcLANQQYFggAJgIb
+AhYhBLHSvRN1vst4TPT4xNc89jjFPAa+BQJl/YgrBQkLehCqAIEJENc89jjFPAa+
+diAEGRYIAB0WIQSjzJyHC50xCrrUzy9RcisI/kdFogUCXJLPgQAKCRBRcisI/kdF
+oqdMAQCgH45aseZgIrwKOvUOA9QfsmeE8GZHYNuFHmM9FEQS6AD6A4x5aYvoY6lo
+98pgtw2HPDhmcCXFItjXCrV4A0GmJA4ZxgD/bphBjTlsu2deIiNOIDOsZCtFfFMc
+8DwC/tMMUe8h61YA/2wdKhTw59PCZOIScbAuWxXtAlyVmDdg3P9ilyn1OpAGwsA1
+BBgWCAAmAhsCFiEEsdK9E3W+y3hM9PjE1zz2OMU8Br4FAmTyc5wFCQl8aBsAgQkQ
+1zz2OMU8Br52IAQZFggAHRYhBKPMnIcLnTEKutTPL1FyKwj+R0WiBQJcks+BAAoJ
+EFFyKwj+R0Wip0wBAKAfjlqx5mAivAo69Q4D1B+yZ4TwZkdg24UeYz0URBLoAPoD
+jHlpi+hjqWj3ymC3DYc8OGZwJcUi2NcKtXgDQaYkDq8mAPoD2rY6vX3V/dxmpP/L
+legQkygGhHRGgSJqfXrI5nOWfgEAwYoVw4TDz5dBHMghYdUG3lOs3nNjdS0D4zWC
+fHZ5HwvCwDUEGBYIACYCGwIWIQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUCYzxUmQUJ
+CHbyGACBCRDXPPY4xTwGvnYgBBkWCAAdFiEEo8ychwudMQq61M8vUXIrCP5HRaIF
+AlySz4EACgkQUXIrCP5HRaKnTAEAoB+OWrHmYCK8Cjr1DgPUH7JnhPBmR2DbhR5j
+PRREEugA+gOMeWmL6GOpaPfKYLcNhzw4ZnAlxSLY1wq1eANBpiQOaukA/0gjmfw6
+5eZaic93tHkNOblWJNW8/A5ohLC1TFCfy6gLAQDsxknT9a3CVpYs6sYIRPdjYbcE
+e2BOBCnkOme1/PZhB8LANQQYFggAJgIbAhYhBLHSvRN1vst4TPT4xNc89jjFPAa+
+BQJiOJ5fBQkGwpLeAIEJENc89jjFPAa+diAEGRYIAB0WIQSjzJyHC50xCrrUzy9R
+cisI/kdFogUCXJLPgQAKCRBRcisI/kdFoqdMAQCgH45aseZgIrwKOvUOA9QfsmeE
+8GZHYNuFHmM9FEQS6AD6A4x5aYvoY6lo98pgtw2HPDhmcCXFItjXCrV4A0GmJA4Q
+EgD/Ss+vMELdwxMiFuJLWiOnnsrLzPd3R8ILPqfP4/JInO0A/2103C6rNCfrX04/
+7TeAluj3CZMd3QyXzbxgGN/L1VIMwsDaBBgWCAAmAhsCFiEEsdK9E3W+y3hM9PjE
+1zz2OMU8Br4FAmD3wPoFCQXwc3kBJgkQ1zz2OMU8Br52IAQZFggAHRYhBKPMnIcL
+nTEKutTPL1FyKwj+R0WiBQJcks+BAAoJEFFyKwj+R0Wip0wBAKAfjlqx5mAivAo6
+9Q4D1B+yZ4TwZkdg24UeYz0URBLoAPoDjHlpi+hjqWj3ymC3DYc8OGZwJcUi2NcK
+tXgDQaYkDo0gBBkWCAAdFiEEo8ychwudMQq61M8vUXIrCP5HRaIFAlySz4EAIQkQ
+UXIrCP5HRaIWIQSjzJyHC50xCrrUzy9RcisI/kdFoqdMAQCgH45aseZgIrwKOvUO
+A9QfsmeE8GZHYNuFHmM9FEQS6AD6A4x5aYvoY6lo98pgtw2HPDhmcCXFItjXCrV4
+A0GmJA4WIQSx0r0Tdb7LeEz0+MTXPPY4xTwGviVSAQCYzxoFUgE63ZGSHiqoQzYM
+AadrBx4omVuzdMBMOuLZuAEAonvbinwCBNofi3hlVmeZX6qFzb4aBP/GFrDVXcrE
+OwzCwDUEGBYIACYCGwIWIQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUCX+IphwUJBI5l
+BgCBCRDXPPY4xTwGvnYgBBkWCAAdFiEEo8ychwudMQq61M8vUXIrCP5HRaIFAlyS
+z4EACgkQUXIrCP5HRaKnTAEAoB+OWrHmYCK8Cjr1DgPUH7JnhPBmR2DbhR5jPRRE
+EugA+gOMeWmL6GOpaPfKYLcNhzw4ZnAlxSLY1wq1eANBpiQOcogBAOhHv6bQKzrt
+qBnNjpKn9/DHzfau0hU583eb7rQCOCG9AQCoaHQM/I9hzFKcTVRozxxMHKOxdX5J
+Ef+/BiRbIPqgA8LANQQYFggAJgIbAhYhBLHSvRN1vst4TPT4xNc89jjFPAa+BQJe
+zg1MBQkDekjLAIEJENc89jjFPAa+diAEGRYIAB0WIQSjzJyHC50xCrrUzy9RcisI
+/kdFogUCXJLPgQAKCRBRcisI/kdFoqdMAQCgH45aseZgIrwKOvUOA9QfsmeE8GZH
+YNuFHmM9FEQS6AD6A4x5aYvoY6lo98pgtw2HPDhmcCXFItjXCrV4A0GmJA7kIgD8
+CpygG83oGkgE8TdXHRYmuJvNLiTRRzEC+u/VkgRtxSABAPLi4MRrnekqhYCGhUeM
+K86yIM1Vqh8Q7ruenyUn/4gHwsA1BBgWCAAmAhsCFiEEsdK9E3W+y3hM9PjE1zz2
+OMU8Br4FAl2wSBIFCQI6PJEAgQkQ1zz2OMU8Br52IAQZFggAHRYhBKPMnIcLnTEK
+utTPL1FyKwj+R0WiBQJcks+BAAoJEFFyKwj+R0Wip0wBAKAfjlqx5mAivAo69Q4D
+1B+yZ4TwZkdg24UeYz0URBLoAPoDjHlpi+hjqWj3ymC3DYc8OGZwJcUi2NcKtXgD
+QaYkDrXuAP9o7uEtR63HZwFcnfgSuYnpXaYDiaiqfRBbo3/syrqCrAD+LqvPrwXO
+kJla4vhMUm6k/6uf6XFHAZMQVXXWuhvvsgzCwDUEGBYIACYWIQSx0r0Tdb7LeEz0
++MTXPPY4xTwGvgUCXJLPgQIbAgUJARzEAACBCRDXPPY4xTwGvnYgBBkWCAAdFiEE
+o8ychwudMQq61M8vUXIrCP5HRaIFAlySz4EACgkQUXIrCP5HRaKnTAEAoB+OWrHm
+YCK8Cjr1DgPUH7JnhPBmR2DbhR5jPRREEugA+gOMeWmL6GOpaPfKYLcNhzw4ZnAl
+xSLY1wq1eANBpiQOCi4BALk/32S/of48v0Fvcn0ouAU+ejxlO5/zKFM4zsBvH5Y+
+AQDgpvn6i4azNEkM84wNKnQq7Eyv+6wUmfdPLWzWDmsIDs44BFySz2oSCisGAQQB
+l1UBBQEBB0AxlRumDW6nZY7A+VCfek9VpEx6PJmdJyYPt3lNHMd6HAMBCAfCfgQY
+FggAJgIbDBYhBLHSvRN1vst4TPT4xNc89jjFPAa+BQJp4JbXBQkOaottAAoJENc8
+9jjFPAa+RNUA/2faQO/nFT06E+MlhlQdo/0chlQXC5TZMPTVvVBFwoLOAP9xLJK0
+ow5EjTzYJB4K810AL/Iv6PEOAEgA4cPTHVlbCcJ+BBgWCAAmAhsMFiEEsdK9E3W+
+y3hM9PjE1zz2OMU8Br4FAmfRdNIFCQ1kZugACgkQ1zz2OMU8Br7QzQEA89GrveQu
+cdg0MyeJUE7NR2LZd4XfaJ2MRZZ+O/IglQ4A/jBn4AvEh6GKzdTDYHMZkTq21dDZ
+D4N4Ik7dM+kippcMwn4EGBYIACYCGwwWIQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUC
+Zf2IMAUJC3oQxgAKCRDXPPY4xTwGvqYaAP46w6iK5f+eb1DAYSEL/kz+t+Xdogkx
+J8hZm1HD/foIegD6A7F2pVXdnDhsULUgvkxfD5GTJ9Uy5CSgI1ca/sceNALCfgQY
+FggAJgIbDBYhBLHSvRN1vst4TPT4xNc89jjFPAa+BQJk8nOjBQkJfGg5AAoJENc8
+9jjFPAa+yXkA/A1Fk7LxXpL602+YBaL7qSqTSaNBt4PhUvcEPJu2WbskAQCKi0mn
+aQh1LRtxYyUtU8Z3Rc9ejxORaJAkj+lB157JAcJ+BBgWCAAmAhsMFiEEsdK9E3W+
+y3hM9PjE1zz2OMU8Br4FAmM8VI8FCQh28iUACgkQ1zz2OMU8Br4/fQD+Jk5tOvUO
+mp1t3JBH7BZZzYGy6ajY9ASevhXCfPFWakIA/36XzijMCteLI8GvpvLrU9nSO/9J
+IQeeYtuOvvUIvRcMwn4EGBYIACYCGwwWIQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUC
+YjieVAUJBsKS6gAKCRDXPPY4xTwGvtBeAQCF3zU75pqfawuyUaMLvj2qdqoAfBqf
+03HPx/4n1LMbdQEA9Mw5HhqgL67wrFTGQ1TncJx/5uAYdjg4nSBMMsuk8gbClQQY
+FggAJgIbDBYhBLHSvRN1vst4TPT4xNc89jjFPAa+BQJg98DxBQkF8HOHACEJENc8
+9jjFPAa+FiEEsdK9E3W+y3hM9PjE1zz2OMU8Br7UPwD8D/F8Q1BKIIl1iGrYOAPK
+GqduNgiUIz3/SK8BExn+3JABAKBf1wqCTTrEikRj4E51QO7w/LTUKb+gWbJOFgd0
+oTgHwn4EGBYIACYCGwwWIQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUCX+IpewUJBI5l
+EQAKCRDXPPY4xTwGvuNdAQDitCEggwuEXfyy1ATEn2bCyKSNcrhZwz6fjgHNBHPb
+ZAEAydSc/15cQsH/4GKKky15GcRs3d90Fox9iSlWcijRdgrCfgQYFggAJgIbDBYh
+BLHSvRN1vst4TPT4xNc89jjFPAa+BQJezg1ABQkDekjWAAoJENc89jjFPAa+7xAA
+/0ufXfqEu3/It/HgpiMHE8B1932hFh3fGpYUHE1maf4GAP9dmuE1pjcwQFXjIjHi
+2zolPt0l9cZxs2PErVPYGVd1AcJ+BBgWCAAmAhsMFiEEsdK9E3W+y3hM9PjE1zz2
+OMU8Br4FAl2wSAYFCQI6PJwACgkQ1zz2OMU8Br7oQAEA720m68QcJqr2xMCbqaz1
+Z/Tcf28H1in20tg6oP4bjoQA/RNWAZsa+aogGxSSj9Du0o9th11+y4gC8n/HjkXo
+STwIwn4EGBYIACYWIQSx0r0Tdb7LeEz0+MTXPPY4xTwGvgUCXJLPagIbDAUJARzE
+AAAKCRDXPPY4xTwGvuJDAQCCxva8K4XD0gL2suFw4CfEqgx5vALCgNxO84q8VauV
+/QEAjZVwfkVoe8l+tbrZCa84gakLYv497xyl+nN8lOBSzQs=
+=s5gp
 -----END PGP PUBLIC KEY BLOCK-----
diff --git a/libidn2.spec b/libidn2.spec
index e505740..5d98347 100644
--- a/libidn2.spec
+++ b/libidn2.spec
@@ -1,7 +1,7 @@
 Summary:          Library to support IDNA2008 internationalized domain names
 Name:             libidn2
 Version:          2.3.8
-Release:          4%{?dist}
+Release:          5%{?dist}
 License:          (GPL-2.0-or-later OR LGPL-3.0-or-later) AND GPL-3.0-or-later
 URL:              https://www.gnu.org/software/libidn/#libidn2
 
@@ -15,6 +15,7 @@ BuildRequires:    make
 BuildRequires:    gettext
 BuildRequires:    libunistring-devel
 BuildRequires:    texinfo
+BuildRequires:    openpgpverify
 Provides:         bundled(gnulib)
 
 %description
@@ -40,7 +41,7 @@ The idn2 package contains the idn2 command line tool for testing
 IDNA2008 conversions.
 
 %prep
-%{gpgverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
+%{openpgpverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
 %setup -q
 
 %build
@@ -87,6 +88,9 @@ rm -f $RPM_BUILD_ROOT%{_datadir}/info/dir
 %{_infodir}/%{name}.info*
 
 %changelog
+* Mon Oct 12 2026 Jakub Jelen <jjelen@redhat.com> - 2.3.8-5
+- Verify upstream source signatures with openpgpverify
+
 * Thu Jul 16 2026 Fedora Release Engineering <releng@fedoraproject.org> - 2.3.8-4
 - Rebuilt for https://fedoraproject.org/wiki/Fedora_45_Mass_Rebuild
 
-- 
2.55.0

